src/utils/config.py

```diff
@@ -29,12 +30,16 @@
 BRAZE_REST_ENDPOINT=os.getenv('BRAZE_REST_ENDPOINT')
 
 SNOWFLAKE_DATA_RETENTION_CONNECTION_DICT = {
-    "user_env_var_name": "SNOWFLAKE_DATA_RETENTION_USER",
-    "private_key_env_var_name": "SNOWFLAKE_DATA_RETENTION_PRIVATE_KEY",
-    "role_env_var_name": "SNOWFLAKE_DATA_RETENTION_ROLE",
-    "warehouse_env_var_name": "SNOWFLAKE_DATA_RETENTION_WAREHOUSE",
-    "account_env_var_name": "SNOWFLAKE_ACCOUNT",
+    "account": os.getenv("SNOWFLAKE_ACCOUNT"),
+    "user": os.getenv("SNOWFLAKE_USER"),
+    "private_key": base64.b64decode(os.getenv("SNOWFLAKE_PRIVATE_KEY")),
+    "warehouse": os.getenv("SNOWFLAKE_WAREHOUSE"),
+    "role": os.getenv("SNOWFLAKE_DATA_RETENTION_ROLE"),
+    "database": os.getenv('SNOWFLAKE_DATA_RETENTION_DB'),
+    "schema": os.getenv('SNOWFLAKE_DATA_RETENTION_SCHEMA')
 }
 
-SNOWFLAKE_DATA_RETENTION_DB = os.getenv('SNOWFLAKE_DATA_RETENTION_DB')
-SNOWFLAKE_DATA_RETENTION_SCHEMA = os.getenv('SNOWFLAKE_DATA_RETENTION_SCHEMA')
+SNOWFLAKE_SNOWPLOW_DB = os.getenv('SNOWFLAKE_SNOWPLOW_DB')
+SNOWFLAKE_SNOWPLOW_SCHEMA = os.getenv('SNOWFLAKE_SNOWPLOW_SCHEMA')
+SNOWFLAKE_RAWDATA_DB = os.getenv('SNOWFLAKE_RAWDATA_DB')
```

Comment:
It looks like new environment variables were added. We will have to add these to the ECS environment.

Reply:
@gkatre I don't see `SNOWFLAKE_RAWDATA_DB` in [.aws/src/config/index.ts](https://github.com/Pocket/data-flows/pull/84/files#diff-562941ffdeea75262c91bd9b45ffaf2093301e296762d4a32e5feee4f1b35ba8) yet.